### PR TITLE
fix: reset password bug

### PR DESCRIPTION
**Pull request by Sudhanva-Nadiger**
#### Summary
- Users on both the Mattermost public version and beta version are receiving an error message: "Unable to reset your password". However, at least for the beta version, the email sends successfuly but the error is still shown. On the mobile web app version and on desktop, no issue is present.

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-50501

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on:  Realme 6 pro

#### Screenshots
https://user-images.githubusercontent.com/93595710/219104712-5a2205e9-7820-47e8-8e27-d0907279598b.mp4

#### Release Note

```release-note
Fix "Unable to reset your password"
```

### CODE REVIEW

**Review by enahum — APPROVED** [review]

File: app/actions/remote/session.ts [hunk 1]
```diff
@@ -245,7 +245,7 @@ export const sendPasswordResetEmail = async (serverUrl: string, email: string) =
         return {error};
     }
     return {
-        data: response.data,
+        data: response.status,
```
Comment (larkox) [inline app/actions/remote/session.ts:248]: If what we are returning and checking is the status, I would also change the name of the variable, as:
```suggestion
        status: response.status,
```

**Review by larkox — CHANGES_REQUESTED** [review]
Great work! Just one minor change.

**Review by larkox — APPROVED** [review]

**Review by enahum — APPROVED** [review]

**Comment by Sudhanva-Nadiger** [comment]
Thanks @enahum and @larkox, It was my first contribution to matter most, I am very happy, Looking forward to learn contribute more in future 🤍

**Comment by enahum** [comment]
Nicely done @Sudhanva-Nadiger is great to have you as a contributor of Mattermost. yay!!!

**Pull request merged**